gymnasium/vector/vector_env.py

```diff
@@ -10,14 +10,35 @@
 
 
 class VectorEnv(gym.Env):
-    """Base class for vectorized environments. Runs multiple independent copies of the same environment in parallel.
+    """Base class for vectorized environments to runs multiple independent copies of the same environment in parallel.
 
-    This is not the same as 1 environment that has multiple subcomponents, but it is many copies of the same base env.
+    This can provide a linear speed-up in the steps taken per second given the number of sub-environments used.
+    When a sub-environment terminates or truncates, the sub-environment is automatically reset with the final step
+    and info including in the returned info in `"final_observation"` and `"final_info"`. See :meth:`step` for more information.
```

Comment:
I think there's a typo here, not too sure what this is trying to say.

Reply:
Updated